docs/04-logging.md

```diff
@@ -59,6 +58,58 @@ checksum can only be computed when Python is running in non-interactive mode,
 so if you are following along this tutorial via the Python shell, you won't
 see a checksum in your log file.
 
+## Logging return values
+
+PlanOut provides the ability to dynamically disable exposure logging by returning `False` from the assignment function. There are many cases where this may be useful, both when initially creating your experiment and when iterating on it.
+
+
+### Example use when initially creating an experiment
+
+For instance, suppose you are running an experiment on a new feature and want to only target users in Germany for your experiment. By returning `False` from your assignment function for users that are not from Germany, PlanOut ensures that you will only have exposure logs for users that are from Germany and can still fix parameters for the users that are not from Germany. In this case your `assign` function may look something like this if it pulls the current user's country from an external service:
+
+```python
+class Exp1(SimpleExperiment):
+  def assign(self, params, userid):
+    country = get_user_country(userid)
+    if country != 'Germany':
+      params.feature_enabled = false
```

Comment:
This is a nice example, but FWIW, it's better style to define all the defaults for your params at the top of your script.